examples/src/examples-ssr.js

```diff
@@ -22,4 +22,4 @@ export default function() {
             </body>
         </html>
     `;
-};
+}
```

Comment:
is there really not supposed to be a semicolon here? I guess I don't know the semantics of `export default` terribly well.